config.schema.json

```diff
@@ -78,6 +78,12 @@
               "description": "Can be found in your platform app (device info -> virtual ID) and is exposed during plugin boot (use the Tuya ID)",
               "required": true
             },
+            "max_brightness": {
+              "title": "The Device max brigthness",
+              "type": "number",
+              "default": "255",
+              "description": "Each dimmer light has different max value, find it and set it here"
```

Comment:
Please document this in the readme file, specifically how people should find this.

Probably by using the Tuya app to set the value to max and using the API to check the current value?

Reply:
@milo526 Let me know if the description is good enough

Reply:
See the provided feedback in the relevant file.